src/register/registrationFields/HonorCode.jsx

```diff
@@ -30,12 +30,22 @@ const HonorCode = (props) => {
           values={{
             platformName: getConfig().SITE_NAME,
             tosAndHonorCode: (
-              <Hyperlink variant="muted" destination={getConfig().TOS_AND_HONOR_CODE || '#'} target="_blank">
+              <Hyperlink
```

Comment:
Isn't the `variant` attribute not needed now?

Reply:
nope, we want to make it visible and different from normal text